src/cluecode/finder_data.py

```diff
@@ -223,10 +243,13 @@ def classify(s, data_set, suffixes=None):
 
 classify_host = partial(classify, data_set=JUNK_HOSTS_AND_DOMAINS, suffixes=JUNK_DOMAIN_SUFFIXES)
 
-classify_email = partial(classify, data_set=JUNK_EMAILS, suffixes=JUNK_DOMAIN_SUFFIXES)
+classify_email = partial(classify, data_set=JUNK_EMAILS, suffixes=JUNK_DOMAIN_SUFFIXES, exact_match=JUNK_EXACT_DOMAINS)
 
 
 def classify_url(url):
+    '''
```

Comment:
Please use double quotes for docstrings.

Reply:
Noted, thanks!